ui/pages/[appName].jsx

```diff
@@ -0,0 +1,350 @@
+import { useRouter } from "next/router";
+import { useQuery } from "react-query";
+import { useMemo } from "react";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import MaUTable from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import { useTable, useFlexLayout } from "react-table";
+import TableContainer from "@material-ui/core/TableContainer";
+import Paper from "@material-ui/core/Paper";
+import { css } from "@emotion/react";
+import Link from "next/link";
+
+async function fetchStuff(url) {
+  try {
+    const response = await fetch(url);
+    return response.json();
+  } catch (err) {
+    throw new Error("Network response was not ok");
+  }
+}
+
+const App = () => {
+  const router = useRouter();
+  const { appName } = router.query;
+
+  const hierarchy = [
+    "global",
+    "app-global",
+    "environment-class",
+    "app-environment-class",
+    "environment",
+    "app",
+  ];
+
+  const { status, data, error, isFetching } = useQuery(["stuff", appName], () =>
+    fetchStuff(`http://localhost:3000/test-data/${appName}.json`)
+  );
+
+  const {
+    status: appFetchStatus,
+    data: apps,
+    error: appFetchError,
+    isFetching: isFetchingApps,
+  } = useQuery(["apps"], () =>
+    fetchStuff(`http://localhost:3000/test-data/applications.json`)
+  );
```

Comment:
For local development can we move these into local json files directly so that we can get enable preview environments? e.g. https://github.com/flanksource/karina-ui/blob/master/src/App.vue#L60

Reply:
Can we task it out in a ticket and defer to a later PR? I don't have test data that isn't client data yet.